dashboard/head.py

```diff
@@ -17,7 +17,11 @@
 from ray._private.gcs_utils import GcsClient, GcsAioClient, check_health
 from ray.dashboard.datacenter import DataOrganizer
 from ray.dashboard.utils import async_loop_forever
-from ray.dashboard.consts import DASHBOARD_METRIC_PORT
+from ray.dashboard.consts import (
+    DASHBOARD_METRIC_PORT,
```

Comment:
@rkooo567 what's the difference between `DASHBOARD_METRIC_PORT` and `--metrics-export-port`?

Reply:
@jjyao if I understand correctly, DASHBOARD_METRIC_PORT is specifically for exposing Dashboard metrics to Prometheus. `--metrics-export-port` exposes more general metrics from the node (and is common to the head and workers).